Assets/_Scripts/GameStatus.cs

```diff
@@ -302,17 +304,33 @@ IEnumerator RebornPlayerWithID(int id)
 	/// <param name="prefab">Prefab.</param>
 	/// <param name="Position">Position.</param>
 	/// <param name="mat">Mat.</param>
-	GameObject InstantiateWizardInstance(GameObject prefab, Vector3 Position, Material mat) {
+	GameObject InstantiateWizardInstance(GameObject prefab, GameObject modelPrefab, Vector3 Position, Material mat) {
```

Comment:
When changing the prototype, please update the auto-generated comment and add necessary information as well.
I won't understand what the prefab means when there are two prefab. Please name it properly, say controllerPrefab?